specification/sql/resource-manager/common/v1/types.json

```diff
@@ -110,11 +110,13 @@
         "type": {
           "description": "The identity type. Set this to 'SystemAssigned' in order to automatically create and assign an Azure Active Directory principal for the resource.",
           "enum": [
-            "SystemAssigned"
+            "None",
+            "SystemAssigned",
+            "UserAssigned"
           ],
           "type": "string",
           "x-ms-enum": {
-            "name": "IdentityType",
+            "name": "IdentityTypeWithNone",
```

Comment:
Can we keep this name? as it may cause sdk breaking change

Reply:
updated in the new iteration. feel free to check.